examples/active.ini

```diff
@@ -1,21 +1,24 @@
 [acq_params]
 amplitude = 1.0
-p90_us = 4.8
```

Comment:
I changed the base branch so it merges to master.  In general, before merging the PR, we should just checkout active.ini from master, unless there's an exceptional reason to change something